internal/pkg/coordinator/monitor.go

```diff
@@ -486,7 +486,8 @@ func runUnenrollerWork(ctx context.Context, bulker bulk.Bulk, policyId string, u
 
 	agentIds := make([]string, len(agents))
 
-	for i, agent := range agents {
+	for i := range agents {
+		agent := agents[i]
```

Comment:
good one

Reply:
The linter complained at me, the address of `agent` for `_, agent := range agents` is the same, and we pass the address below